src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleRules = sourceString.replace(/;\s*;/g, ' ').trim().split(';');
```

Comment:
The use of `replace` here to deal with consecutive semicolons is not necessary, as the task does not require handling multiple semicolons without any styles between them. Additionally, the chaining of `trim()` and `split()` should follow the indentation rule for better readability. Consider refactoring this line to maintain readability and avoid unnecessary operations.